### DISCUSSION TITLE
Add an option to open a new instance of emacs by control+click on the emacs icon.

### DISCUSSION THREAD

**Discussion by sudo-human** [d12frosted/homebrew-emacs-plus#633] [General]
When clicking on the Emacs.app icon, it opens an emacs instance. But if I click on it again, it focuses the same instance.
I added the icon to the dock and then control + clicked it, it only has `New Frame` as the option and no option to launch a new instance.

<img width="384" alt="Screenshot 2023-12-07 at 11 24 32 PM" src="https://github.com/d12frosted/homebrew-emacs-plus/assets/13881173/1f6d25fd-8db2-4961-8062-65b0f2fa3316">

How can we include another option to open a new instance in emacs? I'm very new to Mac and fairly new to emacs.

**Comment by d12frosted** [comment]
> But if I click on it again, it focuses the same instance.

This is how macos applications work. There are some apps that hack around it. One of the common ways is to run emacs not via `Emacs.app`, but rather via script (ie`emacs` executable which is bundled with Emacs+). There are few options that come to mind (:

- Bind `emacs` executable to some key using [skhd](https://github.com/koekeishiya/skhd). This is how I do it althgouh not for running multiple instances, I just want to have a way to quickly run Emacs :) 
- If you need it on your Dock or whatever, you can create an app using Automation.app (bundled with macos) - just to run `emacs` executable.

Since it's not a feature request, nor a bug in Emacs+ (which is just a formula for homebrew), I am converting this into discussion.
